platforms/mobile/ios.md

```diff
@@ -1 +1,14 @@
-# Langley Foxall iOS Apps
\ No newline at end of file
+# Langley Foxall iOS
+
+iOS is a big part of the company and it's what got us started. We write our iOS apps in purely Swift.
```

Comment:
Some cases require the use of Obj-C or Obj-C++, we do mainly use Swift as you say however and projects wouldn't be entirely Obj-C. Might be worth mentioning that.